pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -316,3 +323,14 @@ func (r *Client) cacert() []byte {
 	}
 	return nil
 }
+
+//
+// Get the insecureSkipVerify boolean value from the ovirt connection secret.
+func (r *Client) insecureSkipVerify() (bool, error) {
```

Comment:
> You are not returning errors at any point of the function.

strconv.ParseBool returns (bool, error)

Reply:
ahhh in that case I would do it as in the settings
https://github.com/konveyor/forklift-controller/blob/367d2e9e2378ba7419881aff80fd32472a4b8538/pkg/settings/settings.go#L96-L106

Reply:
> You are not returning errors at any point of the function.

yes, strconv.ParseBool returns (bool, error)

> Instead of having the same function defined in 2 places maybe move it to some shared lib like https://github.com/konveyor/controller

I'm not sure it's a functionality we neen in `konveyot/controller`, maybe we have a better place to store small common functionalities in this repo, @mansam WDYT ?

Reply:
> Instead of having the same function defined in 2 places maybe move it to some shared lib 

@mnecas I refactored the 3 `insecureSkipVerify` methods into the ovirt pkg, PTAL